firmware/application/ui_navigation.cpp

```diff
@@ -327,6 +327,27 @@ SystemStatusView::SystemStatusView(
         refresh();
     };
 
+    toggle_fake_brightness.on_change = [this, &nav](bool v) {
+        set_dirty();
+        pmem::set_apply_fake_brightness(v);
+        if (nav.is_valid() && v) {
+            nav.display_modal(
+                "Brightness",
+                "You just enabled brightness\n"
+                "adjust.\n"
+                "performance will be\n"
+                "influented a little");
+
+            // TODO: refresh interface to prevent reboot requirement
+            // TODO: increase performance
+        } else if (!v) {
+            nav.display_modal(
+                "Brightness",
+                "brightness adjust disabled.");
```

Comment:
In my opinion, I would capitalize "brightness", i.e. "Brightness adjust disabled."

Reply:
Thanks! fixed